src/main/java/frc/robot/subsystems/AlgaeGripper.java

```diff
@@ -10,16 +11,19 @@
 import frc.robot.RobotMap;
 
 public class AlgaeGripper extends SubsystemBase {
-
      private static final double COLLECT_SPEED = 0.8;
      private static final double RELEASE_SPEED = -0.5;
      private static final double HOLD_SPEED = 0.2;
-
      private final SparkMax motor;
+     private final RelativeEncoder encoder;
      private final DigitalInput digitalInput;
 
+
      public AlgaeGripper(){
           this.motor = new SparkMax(RobotMap.ALGAE_GRIPPER_MOTOR, SparkLowLevel.MotorType.kBrushless);
+
+          encoder = this.motor.getAlternateEncoder();
```

Comment:
```suggestion
          encoder = this.motor.getEncoder();
```
there's no alternate encoder, you are probably trying to use the integrated encoder

Reply:
Weird I was trying to do getEncoder but it didn't show up